### DISCUSSION TITLE
Add schedule

### DISCUSSION THREAD

**Discussion by btjsgg** [mhaeuser/Battery-Toolkit#29] [Ideas]
Hi, like your program. It would be nice if you could configure a schedule for when the battery needs to be at 100% charged before I go to school and kept at 80% charged at other times.

Thanks in advance,
Jonathan

**Comment by mhaeuser** [comment]
Sounds good to me. However, it would probably be near impossible for Battery Toolkit to reliably determine how long it would take to charge up, so a “charging deadline” would be much harder to implement than a “charging start”. Maybe RE’ing how Apple does it would reveal some private API that could still be used for this.

I’m not able to spend significant time on this tool right now though, so don’t expect progress soon either way.

**Comment by mhaeuser** [comment]
Now tracked at https://github.com/mhaeuser/Battery-Toolkit/issues/56
